client/connect-account-page/test/index.js

```diff
@@ -13,7 +13,11 @@ describe( 'ConnectAccountPage', () => {
 	beforeEach( () => {
 		window.location.assign = jest.fn();
 		global.wcpaySettings = {
-			connectUrl: '/wcpay-connect-url',
+			connect: {
+				url: '/wcpay-connect-url',
+				country: 'US',
+				availableCountries: { US: 'United States (US)' },
+			},
```

Comment:
In this file, we can add tests for changes made to `handleSetup` and check for the newly added behavior of the connect method.

Reply:
That's a good idea, will add. Thanks!

Reply:
Added in f22cec99b75140a95227c44b555e7a9fab5f7600